<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myTabPage</class>
 <widget class="QWidget" name="myTabPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="commonBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>389</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Common</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Common Signal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CommonSignal"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Common Signal Length</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="COMMON_SIGNAL_LENGTH"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Command Signal Length</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="CMD_SIGNAL_LENGTH"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Devide Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Device_ID"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Model"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timesBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>389</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Timers</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="miscBox">
      <attribute name="label">
       <string>Misc</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
